runtime/libffi/z/ffi64.c

```diff
@@ -766,6 +768,50 @@ ffi_prep_args (unsigned char *stack, extended_cif *ecif)
 
 /*======================== End of Routine ============================*/
 
+/**
+ * Helper functions to know if the given struct needs to be treated for complex
+ * type for float or double
+ */
+unsigned short
```

Comment:
This function and the next should be `static`.

Reply:
Done in https://github.com/eclipse-openj9/openj9/compare/6322f343e9e88af4b64f7d08586a47a7ecb89b32..583867825be4477a8319f21ffb952780631fbeb3